app/konnect/configure/runtime-manager/runtime-groups/declarative-config.md

```diff
@@ -124,7 +144,7 @@ the example with your generated UUID:
 {{site.konnect_saas}}:
 
     ```sh
-    deck konnect diff --konnect-email <email> --konnect-password-file <pass>.txt
+    deck diff --konnect-runtime-group default
```

Comment:
It's not clear which file we are using for diffing the configuration. By default, decK looks for `kong.yaml`, but in L108 we are suggesting to modify a `konnect.yaml` file

Reply:
That's a leftover from `deck konnect`, nice catch. Fixing.